src/components/MyKiva/BadgesSection.vue

```diff
@@ -75,6 +77,14 @@ const visibleBadges = computed(() => {
 const levelCaption = badge => {
 	return `Level ${getActiveTierData(badge).level}/${getBadgeWithVisibleTiers(badge).achievementData.tiers.length}`;
 };
+
+const getBadgeStatus = badge => {
+	const activeTier = getActiveTierData(badge);
+	if (activeTier?.level === badge?.achievementData?.tiers?.length) {
```

Comment:
Should we compare complete dates of the tiers instead of the active tier?

Reply:
`getActiveTierData` will return the last level, if all levels are completed, so we should be fine with this logic.